api/hooks/discordBot/commands/meta/restart.js

```diff
@@ -12,7 +12,7 @@ class Restart extends Commando.Command {
 
   async run(msg) {
 
-    const admins = process.env.CSMM_ADMINS.split(',');
+    const admins = (process.env.CSMM_ADMINS || '').split(',');
```

Comment:
Good thing you noticed this! 

This is duplicated code, would be better if you could refactor this to use `sails.config.custom.adminSteamIds` IMO. You can do a search for that to see some examples